indico/modules/categories/client/js/components/CategoryModeration.jsx

```diff
@@ -103,38 +122,47 @@ function RequestList({requests, onSubmit, loading}) {
   }
 
   return (
-    <Table celled>
+    <Table styleName="table">
       <Table.Header>
         <Table.Row>
           <Table.HeaderCell />
           <Translate as={Table.HeaderCell}>User</Translate>
           <Translate as={Table.HeaderCell}>Event</Translate>
+          <Translate as={Table.HeaderCell}>Event Path</Translate>
+          <Translate as={Table.HeaderCell}>Event Date</Translate>
           <Translate as={Table.HeaderCell}>Requested Date</Translate>
-          <Translate as={Table.HeaderCell}>Comment</Translate>
         </Table.Row>
       </Table.Header>
       <Table.Body>
         {loading ? (
           <PlaceholderTableRow />
         ) : (
-          requests.map(({id, requestor, requestorComment, event, requestedDt}) => (
-            <RequestTableRow
-              key={id}
-              id={id}
-              event={event}
-              requestor={requestor}
-              requestorComment={requestorComment}
-              requestedDt={requestedDt}
-              selected={selected.has(id)}
-              onSelect={() => select(id)}
-            />
+          requests.map(({id, requestor, requestorComment, event, category, requestedDt}) => (
+            <>
+              <RequestTableRow
+                key={id}
+                id={id}
+                event={event}
+                category={category}
+                requestor={requestor}
+                requestedDt={requestedDt}
+                selected={selected.has(id)}
+                onSelect={() => select(id)}
+              />
+              {requestorComment ? (
+                <Table.Row>
+                  <Table.Cell />
+                  <Table.Cell colSpan="5">{requestorComment}</Table.Cell>
+                </Table.Row>
+              ) : null}
```

Comment:
Shouldn't this be part of the `RequestTableRow` component? (ie render either one or two rows from within that component)